dags/taar_amodump.py

```diff
@@ -48,6 +48,21 @@
     dag=dag
 )
 
+editorial_whitelist = EMRSparkOperator(
+    task_id="taar_update_whitelist",
+    job_name="Generate a JSON blob from editorial reviewed addons for TAAR",
+    execution_timeout=timedelta(hours=1),
+    instance_count=1,
+    owner="vng@mozilla.com",
+    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
+    env=mozetl_envvar("taar_update_whitelist",
+                      {},
```

Comment:
This is missing a required `--date` option from here -- I'm not sure whether it should be `ds` or `ds_nodash` though from a cursory glance. 

https://github.com/mozilla/python_mozetl/blob/d684691258d01946f7b58d62aefd414d63419221/mozetl/taar/taar_update_whitelist.py#L75

Reply:
I've added the date in 92bf188